screensharing/main.js

```diff
@@ -85,6 +123,32 @@ class ScreenShareMainHook {
             case SCREEN_SHARE_EVENTS.STOP_SCREEN_SHARE:
                 this._jitsiMeetWindow.webContents.send(SCREEN_SHARE_EVENTS_CHANNEL, { data });
                 break;
+            case SCREEN_SHARE_EVENTS.DO_GDM: {
+                const { callback } = this._gdmData;
+
+                this._gdmData = null;
+
+                if (!data.source) {
+                    callback(null);
+
+                    break;
+                }
+
+                const constraints = {
+                    video: data.source
+                };
+
+                // Setting `audio` to `undefined` throws an exception.
```

Comment:
what does this mean?

I guess we still don't set it in certain cases which will be equivalent to setting it to undefined...

Reply:
It's only supported if it's a string. If absent it's ok. An API quirk.

Reply:
Nice